packages/front/src/routes/association/[identifier]/components/Establishments/Establishments.controller.ts

```diff
@@ -6,21 +7,20 @@ import { currentAssociation, currentAssoSimplifiedEtabs } from "$lib/store/assoc
 export class EstablishmentsController {
     associationStore: Store<AssociationEntity>;
     establishmentsStore: Store<SimplifiedEstablishment[]>;
-    filteredEstablishments: Store<SimplifiedEstablishment[]>;
+    filteredEstablishments: Store<SimplifiedEstablishment[]> = new Store([]);
     visibleEstablishments: Store<SimplifiedEstablishment[]> = new Store([]);
     totalPages: Store<number> = new Store(1);
     currentPage: Store<number> = new Store(1);
+    init = true;
+    unsubscribes: unknown[] = [];
```

Comment:
```suggestion
    isStillLoading = true;
    storeSubscriptions: unknown[] = [];
```
Je trouve ça + clairs comme noms

Reply:
ok pour `isStillLoading` mais `unsubscribes` c'est le terme svelte et c'est bien un tableau de fonction pour se désinscrire des stores

Reply:
Comment ça "c'est le terme svelte" ?

Reply:
ce n'est pas un tableau de subscriptions mais un tableau de méthode unsubscribe.

de toute façon j'ai passé ça en const dans la méthode qui appel `onDestroy`, pas besoin de l'avoir lié à la classe au final